### PR TITLE
closes #6 conect spaces to users

### LINKED ISSUE #6: Connect spaces to users
As a Host, 
So that I can see my spaces,
I want my spaces to be connected to my account

### PR DESCRIPTION
**Pull request by ccfz**
- Side effect was that one now needs to log in to post!
    *Important for when testing space_spec to signup before
  - added spec for the space index page
  - edited space controller
  - edited space and user model to add one to many support

### CODE REVIEW

File: app/controller/spaces.rb [hunk 1]
```diff
@@ -10,7 +11,17 @@ class Bnb < Sinatra::Base
...
+    
+
+    space = Space.new(name: params[:space_name])
+    # if user --> if you want to limit spaces to logged in status
+    
+    space.user= User.get(session[:user_id])
+    space.save
+  
+     
+
```
Comment (kylebuttner) [inline app/controller/spaces.rb:19]: remove unnecessary spacing

File: app/controller/spaces.rb [hunk 2]
```diff
@@ -10,7 +11,17 @@ class Bnb < Sinatra::Base
...
 
   post '/spaces/new' do
-    space = Space.create(name: params[:space_name])
+
+    
+
+    space = Space.new(name: params[:space_name])
+    # if user --> if you want to limit spaces to logged in status
+    
+    space.user= User.get(session[:user_id])
```
Comment (kylebuttner) [inline app/controller/spaces.rb]: space after space.user (style)

**Comment by kylebuttner** [comment]
:shipit:

**Comment by robota-x** [comment]
:+1:

**Pull request merged**